lib/Sema/TypeCheckGeneric.cpp

```diff
@@ -816,33 +816,33 @@ TypeChecker::validateGenericFuncSignature(AbstractFunctionDecl *func) {
 void TypeChecker::configureInterfaceType(AbstractFunctionDecl *func,
                                          GenericSignature *sig) {
   Type funcTy;
-  Type initFuncTy;
-
+  Type initFuncTy = Type();
+  
   if (auto fn = dyn_cast<FuncDecl>(func)) {
     funcTy = fn->getBodyResultTypeLoc().getType();
     if (!funcTy)
       funcTy = TupleType::getEmpty(Context);
-
+    
   } else if (auto ctor = dyn_cast<ConstructorDecl>(func)) {
     auto *dc = ctor->getDeclContext();
-
+    
     funcTy = dc->getSelfInterfaceType();
     if (!funcTy)
       funcTy = ErrorType::get(Context);
-
+    
```

Comment:
Okay, there's even more below. Please try not to have spurious diffs like this, they're generally straightforward to avoid.